src/Pact/Native/Capabilities.hs

```diff
@@ -143,6 +146,7 @@ createUserGuard :: NativeDef
 createUserGuard =
   defRNative "create-user-guard" createUserGuard'
   (funType (tTyGuard (Just GTyUser)) [("data",tvA),("predfun",tTyString)])
+  []
   "Defines a custom guard predicate, where DATA will be passed to PREDFUN at time \
   \of enforcement. PREDFUN is a valid name in the declaring environment. \
   \PREDFUN must refer to a pure function or enforcement will fail at runtime."
```

Comment:
My thinking was that this would be fine for now, since the examples weren't there before. Though I did mean to remove the `Examples` header in this case.